Changed Campus section card

Rehasree opened the pull request.

Rehasree committed:
Update index.html

Changed campus section design

Rehasree committed:
Update style.css

Css for the campus section added

vibhanshujainiiitr commented:
Can you please arrange them in Alphabetical order.
And if possible, add a link to our page on the bottom of the card.
Like,
**IIIT Raichur**
raichur.iiitiansnetwork.com

Or something like that we can show the user that we also have a website for IIIT Raichur.

Rehasree committed:
Update index.html

Small modifications in IIIT card section(added more button which redirects to website link)

Rehasree committed:
Update style.css

changed services section card

Rehasree committed:
Update index.html

changed service section card
